pages/userSettings/userSettings.ejs

```diff
@@ -13,6 +13,59 @@
     <div id="content" class="container">
       <h1 class="mb-4">Settings</h1>
 
+      <div class="card mb-4">
+        <div class="card-header bg-primary text-white d-flex align-items-center">
+          Logged-in user profile information
+          <a href="/pl/logout" class="btn btn-danger btn-sm ml-auto">Logout</a>
+        </div>
+          <table class="table table-sm two-column-description">
```

Comment:
Can you fix up the indentation here? This should be at the same level as the above `<div>`.

Reply:
Wouldn't the linter / formatter catch that?

Reply:
Nope; this is a legacy EJS file.